web/client/components/import/dragZone/enhancers/__tests__/processFiles-test.jsx

```diff
@@ -158,11 +158,7 @@ describe('processFiles enhancer', () => {
                     .switchMap(({ onDrop = () => { } }) => getMapFile().map((file) => onDrop([file]))).ignoreElements()))
         )(createSink(props => {
             expect(props).toExist();
-            if (props.files) {
```

Comment:
I modified the test because accordingly to my changes if the file is containing `projection` field in it and projection value is not presented in the supportedProjections array `const supportedProjections = ConfigUtils.getConfigProp('projectionDefs') || []` the error will be thrown to show the alert message to the user and prevent the red error screen. The test covers a new code.